.kitchen.yml

```diff
@@ -0,0 +1,32 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  require_chef_omnibus: 12.9
+
+platforms:
+  - name: centos-7.2
+  - name: debian-8.4
+  - name: windows
+    driver:
+      box: opentable/win-2012r2-standard-amd64-nocm
+  - name: centos-7.2-compat
+    provisioner:
+      require_chef_omnibus: 12.4
+    driver:
+      box: bento/centos-7.2
+
+suites:
+  - name: install
+    run_list:
+      - recipe[cerner_splunk_ingredient_test::splunk]
+      - recipe[cerner_splunk_ingredient_test::universal_forwarder]
+  - name: uninstall
```

Comment:
No, that's a good call.  I just didn't think about adding the recipes to the same suite.